libs/react-numerics/src/lib/formatted-input.tsx

```diff
@@ -63,4 +65,6 @@ export interface FormattedInputProps
   onChange: (value: string, changeType: ChangeType) => void;
   /** Pass a handler to be notified of key down events. */
   onKeyDown?: (evt: React.KeyboardEvent<HTMLInputElement>) => void;
+  /** Pass a reference back up to the underlying input element */
+  inputRef?: React.RefObject<HTMLInputElement>;
```

Comment:
My only question would be about `RefObject` vs `MutableRefObject`. Should we accept both? Does one inherit from the other? I'm not fully aware of the differences, I just know in a couple specific places I've had to use one over the other.

Reply:
had a quick convo on the team call, going to leave as is with normal ref, but we can update it to allow mutable version if a use case comes up

Reply:
perfect!